u8timeseries/backtesting/backtesting_udf.py

```diff
@@ -25,8 +28,8 @@ def get_train_val_series(series: TimeSeries, start: pd.Timestamp, nr_points_val:
     :return: a list of (training_set, validation_set) pairs
     """
 
-    assert start in series, 'The provided start timestamp is not in the time series.'
-    assert start != series.end_time(), 'The provided start timestamp is the last timestamp of the time series'
+    assert_log(start in series, 'The provided start timestamp is not in the time series.', logger)
```

Comment:
This is a good idea, however I think here, and in most places where we check user-provided arguments, it would be more appropriate to use `ValueError` exceptions (most of the time).
Until now we were using assert's, but this was a (bad) shortcut, as the exception system is made for this, and will be overall more informative than asserts.

Reply:
done